examples/js/loaders/EXRLoader.js

```diff
@@ -1614,8 +1611,9 @@ THREE.EXRLoader.prototype = Object.assign( Object.create( THREE.DataTextureLoade
 			if ( dwaHeader.rleRawSize > 0 ) {
 
 				var compressed = info.array.slice( inOffset.value, inOffset.value + dwaHeader.rleCompressedSize );
-				var inflate = new Inflate( compressed, { resize: true, verify: true } ); // eslint-disable-line no-undef
-				var rleBuffer = decodeRunLength( inflate.decompress().buffer );
+				var data = fflate.unzlibSync( compressed ); // eslint-disable-line no-undef
+				var acBuffer = new Uint16Array( data.buffer );
```

Comment:
minor thing, `acBuffer` not used here, can remove and switch `rleBuffer` to:
```js
var rleBuffer = decodeRunLength( data.buffer ); ; //line 1615
```

Reply:
not as minor as I though, causes conflict with upper variable name and generates incorrect data.

[current](https://user-images.githubusercontent.com/3927951/103316988-f412d200-4a08-11eb-8c73-80a62ef66c26.png) - [change](https://user-images.githubusercontent.com/3927951/103317025-1c023580-4a09-11eb-981f-96230c18e635.png)

Reply:
Fixed!